async.c

```diff
@@ -502,11 +521,12 @@ static int redisIsSubscribeReply(redisReply *reply) {
 
 void redisProcessCallbacks(redisAsyncContext *ac) {
     redisContext *c = &(ac->c);
-    redisCallback cb = {NULL, NULL, 0, NULL};
+    redisCallback cb = {NULL, NULL, NULL, 0, NULL};
```

Comment:
Since we're placing the new callback at the end.

```suggestion
    redisCallback cb = {NULL, NULL, 0, NULL, NULL};
```

Reply:
`redisCallback cb = {0};` would be fine too.